spec/features/sessions_spec.rb

```diff
@@ -55,6 +55,38 @@
         expect(page).to have_content('Session timed out!')
       end
     end
+
+    describe 'return after timeout' do
+      let(:submit_login_details) do
+        fill_in 'login', with: @user.email
+        fill_in 'password', with: password
+        click_button 'Let me in!'
+      end
+
+      it 'redirects to previous page' do
+        login
+
+        Timecop.freeze(Time.now + 1.hour) do
+          visit new_project_issue_path(Project.find(1))
```

Comment:
`Project.find(1)` will have to change in Pro, so I suggest we store this in a variable like:

```
let (:project) { Project.find(1) }
```

So in Pro, all we have to change is that line.